hintedhandoff_test.py

```diff
@@ -71,6 +71,7 @@ def _do_hinted_handoff(self, node1, node2, enabled, keyspace='ks'):
             else:
                 query_c1c2(session, n, ConsistencyLevel.ONE, tolerate_missing=True, must_be_missing=True)
 
+    @since('3.0', max_version='3.11.x')
```

Comment:
Shouldn't this use `@ported_to_in_jvm`? If we consider that vnodes are not relevant for the test we can also add `@no_vnodes`, this would also skip those tests in 3.0 and 3.11. Anyway, I think that `@no_vnodes` is generally used to skip tests that don't support vnodes, not that don't require them, so runs with and without vnodes are complete.